src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -204,10 +208,24 @@ public void closeMore() {
   }
 
   public void addContent(String content) {
+    waitForElementByElement(textArea);
+    textArea.sendKeys(content);
+    PageObjectLogging.log("addContent", "content was added", true);
+  }
+
+  public void addContentInSourceMode(String content) {
+    waitForElementByElement(sourceModeTextArea);
     sourceModeTextArea.sendKeys(content);
     PageObjectLogging.log("addContent", "content was added", true);
   }
 
+  public String copyContent()
+  {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this left curly brace to the end of previous line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)